templates/jdbc-mysql-repository.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type" : "db",
+            "connectionUrl": "jdbc:mysql://${fcrepo.mysql.host:localhost}:${fcrepo.mysql.port:3306}/fcrepo?createDatabaseIfNotExist=true",
```

Comment:
If we are going to template these files anyways, I'd say we don't set the variables on the JAVA_OPTS and instead just embed them all in here. That way the database password is not visible to anyone on the machine.
So this line becomes
```
"connectionUrl": "jdbc:mysql://{{ fcrepo_db_host }}:{{ fcrepo_db_port }}/{{ fcrepo_db_name }}?createDatabaseIfNotExist=true",
```
Both `fcrepo_db_host` and `fcrepo_db_port` do not exist yet. So we'd need some reasonable defaults defined for both MySQL and PosgreSQL.